composer.json

```diff
@@ -7,14 +7,15 @@
     ],
     "require-dev": {
         "php": ">=8.2",
-        "deptrac/deptrac": "^4.2",
+        "deptrac/deptrac": "^4.4",
         "php-cs-fixer/shim": "^3.91.2",
         "phpstan/phpstan-strict-rules": "^2.0",
         "symfony/console": "^7.4|^8.0",
         "symfony/filesystem": "^7.4|^8.0",
         "symfony/finder": "^7.4|^8.0"
     },
     "minimum-stability": "dev",
+    "prefer-stable": true,
```

Comment:
Had issues with "minimum-stability": "dev" in composer.json, because composer resolves deptrac/deptrac to 4.x-dev instead of the stable 4.4.0. The dev version had a conflict with the bundled PHPStan and php-parser 5.x (the getLexer method was removed in php-parser 5.x).